UPGRADING.md

```diff
@@ -83,6 +83,34 @@ express_app.use('/api', apiRouter);
 // now POST calls to /api/alexa in express will be handled by the app.request() function
 ```
 
+#### Lambda `callback` used by default over `context`
+Since version 4.0.0, the [`callback`](http://docs.aws.amazon.com/lambda/latest/dg/nodejs-prog-model-handler.html#nodejs-prog-model-handler-callback)
```

Comment:
Not a big deal, but since nobody reads these markdown files in markdown format there's no need to line-break them at 80.